util/build_vm.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+# This script is intended to install Mininet and IPMininet into
+# a brand-new Ubuntu virtual machine,
+# to create a fully usable "tutorial" VM.
+set -e
+
+export LC_ALL=C
+
+MN_INSTALL_SCRIPT_REMOTE="https://raw.githubusercontent.com/mininet/mininet/master/util/vm/install-mininet-vm.sh"
```

Comment:
Maybe we should fix a version e.g. `https://github.com/mininet/mininet/blob/2.2.2/util/vm/install-mininet-vm.sh`

Reply:
This is because I would like to add Python 3 compatibility (PR #16) in the near future and only the Mininet master branch is python 3 compatible at the moment.

However, I can add a variable to the script that sets the version of Mininet and let it at "master" for the moment. Then, in the future, fixing to a particular version of Mininet would be easier.

Reply:
:+1: